apps/src/p5lab/spritelab/SpriteLab.js

```diff
@@ -82,8 +82,18 @@ export default class SpriteLab extends P5Lab {
     const current = new Date().getTime();
     if (isPaused) {
       this.library.endPause(current);
+      this.pausedSounds?.forEach(soundUrl =>
+        Sounds.getSingleton().playURL(soundUrl)
+      );
     } else {
       this.library.startPause(current);
+      const soundsById = Sounds.getSingleton().soundsById;
+      this.pausedSounds = Object.keys(soundsById).filter(
+        soundUrl => soundsById[soundUrl].isPlaying_
+      );
+      this.pausedSounds.forEach(soundUrl =>
+        Sounds.getSingleton().stopPlayingURL(soundUrl)
+      );
```

Comment:
i think the sound pausing/restarting functionality should live in the Sounds class. maybe something like `.pauseSounds()` and `.restartPausedSounds()`. we should also make sure when we restart the sounds that we clear out the paused sounds we've stored